### PR TITLE
OCPBUGS-11071: Operator should not add empty strings node arch list i…

### PR DESCRIPTION
**Pull request by TheRealJon**
…n console config

### CODE REVIEW

**Review by spadgett — COMMENTED** [review]
Code LGTM, can we add a unit test?

File: pkg/console/operator/sync_v400.go [hunk 1]
```diff
@@ -621,7 +621,9 @@ func getNodeArchitectures(nodes *corev1.NodeList) []string {
 	nodeArchitecturesSet := sets.NewString()
 	for _, node := range nodes.Items {
 		nodeArch := node.Labels[api.NodeArchitectureLabel]
-		nodeArchitecturesSet.Insert(nodeArch)
+		if nodeArch != "" {
+			nodeArchitecturesSet.Insert(nodeArch)
```
Comment (spadgett) [inline pkg/console/operator/sync_v400.go:625]: Should we log a warning? I don't know if this is ever expected

**Comment by TheRealJon** [comment]
@spadgett I've addressed your comments, PTAL!

File: pkg/console/operator/sync_v400_test.go
Comment (spadgett) [inline pkg/console/operator/sync_v400_test.go:1]: Thanks, looks good. Can we add one more test where multiple nodes have the same arch label (to make sure it only appears in the list once)? This is the most common case.

This isn't new with your fix, but it's a gap in our tests.

**Review by spadgett — COMMENTED** [review]
Thank you for adding tests 👍

**Comment by TheRealJon** [comment]
/label tide/merge-method-squash

@spadgett I added the test case you requested, PTAL!

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**